generate.sh

```diff
@@ -68,6 +68,9 @@ then
 fi
 if [ $2 = 'ruby' ]
 then
+    # https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-skip-certain-files-during-code-generation
+    echo git_push.sh > $1-client/.openapi-generator-ignore
```

Comment:
adding a 
```
mkdir -p $1-client
```
before fix it

Reply:
That makes sense. Using `set -e` would have caught this, should I add that to the script?

Reply:
yes, please!

Reply:
It looks like the script tries to be POSIX shell compatible, but AFAIK `set -e` is a bashism. I'll submit it as a separate PR so we can have the discussion there.